core/src/main/java/io/grpc/internal/ChannelTracer.java

```diff
@@ -121,7 +122,13 @@ void traceOnly(Event event) {
 
   void logOnly(Level logLevel, String msg) {
     if (logger.isLoggable(logLevel)) {
-      logger.log(logLevel, "[" + logId + "] " + msg);
+      LogRecord lr = new LogRecord(logLevel, "[" + logId + "] " + msg);
+      // No resource bundle as gRPC is not localized.
+      lr.setLoggerName(logger.getName());
+      lr.setSourceClassName(ChannelLogger.class.getName());
```

Comment:
Is `ChannelLogger.class.getName()` different from `logger.getName()`? If not, probably cleaner to use the latter.

Reply:
Done.